apps/api/src/database/migrations/2692870736644-migration.ts

```diff
@@ -0,0 +1,237 @@
+            type: 'timestamp',
+            default: 'CURRENT_TIMESTAMP',
+          },
+          {
+            name: 'updated_on',
+            type: 'timestamp',
+            default: 'CURRENT_TIMESTAMP',
+            onUpdate: 'CURRENT_TIMESTAMP',
+          },
+          {
+            name: 'status',
+            type: 'tinyint',
+            default: 1,
+          },
+        ],
+      }),
+    );
+    await queryRunner.createTable(
+      new Table({
+        name: 'notify_providers',
+        columns: [
+          {
+            name: 'provider_id',
+            type: 'int',
+            isPrimary: true,
+            isGenerated: true,
+            isUnique: true,
+          },
+          {
+            name: 'name',
+            type: 'varchar',
+          },
+          {
+            name: 'configuration',
+            type: 'text',
+          },
+          {
+            name: 'application_id',
+            type: 'int',
+          },
+          {
+            name: 'user_id',
+            type: 'int',
+          },
+          {
+            name: 'created_on',
+            type: 'timestamp',
+            default: 'CURRENT_TIMESTAMP',
+          },
+          {
+            name: 'updated_on',
+            type: 'timestamp',
+            default: 'CURRENT_TIMESTAMP',
+            onUpdate: 'CURRENT_TIMESTAMP',
+          },
+          {
+            name: 'status',
+            type: 'tinyint',
+            default: 1,
+          },
+        ],
+      }),
+    );
+    await queryRunner.createTable(
+      new Table({
+        name: 'notify_users',
+        columns: [
+          {
+            name: 'user_id',
+            type: 'int',
+            isPrimary: true,
+            isGenerated: true,
+            isUnique: true,
+            generationStrategy: 'increment',
+          },
+          {
+            name: 'username',
+            type: 'varchar',
+          },
+          {
+            name: 'password',
+            type: 'varchar',
+          },
+          {
+            name: 'role',
+            type: 'tinyint',
+            default: 0,
+          },
+          {
+            name: 'application_id',
+            type: 'int',
+          },
+          {
+            name: 'created_on',
+            type: 'timestamp',
+            default: 'CURRENT_TIMESTAMP',
+          },
+          {
+            name: 'updated_on',
+            type: 'timestamp',
+            default: 'CURRENT_TIMESTAMP',
+            onUpdate: 'CURRENT_TIMESTAMP',
+          },
+          {
+            name: 'status',
+            type: 'tinyint',
+            default: 1,
+          },
+        ],
+      }),
+    );
+    await queryRunner.createTable(
+      new Table({
+        name: 'notify_master_providers',
+        columns: [
+          {
+            name: 'provider_id',
+            type: 'int',
+            isPrimary: true,
+            isGenerated: true,
+            isUnique: true,
+            generationStrategy: 'increment',
```

Comment:
We do not have this in the `notify_providers` table. Is this needed here or missing from there?

Reply:
Assuming we will seed id as well I will remove the increment from here